tests/test_adaption_prompt.py

```diff
@@ -409,6 +774,32 @@ def test_use_cache(self) -> None:
         actual = adapted.generate(input_ids=input_ids, max_length=8)
         assert_close(expected, actual, rtol=0, atol=0)
 
+        #Test Mistral
+        if self.mistral_available:
+            original = MistralForCausalLM(
+                MistralConfig(
+                    vocab_size=16,
+                    hidden_size=8,
+                    intermediate_size=8,
+                    num_hidden_layers=8,
+                    num_attention_heads=4,
+                    num_key_value_heads=2,
+                    use_cache=False,
+                )
+        ).eval()
```

Comment:
Wrong indentation.

Reply:
Thank for pointing out.
Will make it correct!